docs/content/tools/acquia-cli.md

```diff
@@ -29,10 +29,11 @@ fin config set --global SECRET_ACQUIA_CLI_KEY="8842fd3a-e3f3-47e9-916f-6ff760e26
 Refresh `cli` service configuration with `fin project start`. This will inject the values into `cli`
 which will be used for authentication for the Acquia CLI.
 
-Note: It is also possible to add/override these values via `.docksal/docksal-local.env` at the project level. 
```

Comment:
I'd either add an example with `fin config set --env=local ...` or just mention that `--global` can be replaced with `--env=local` to achieve this.